### DISCUSSION TITLE
Running on Orion

### DISCUSSION THREAD

**Discussion by weiwilliam** [dtcenter/METplus#1117] [Existing Builds] [answered]
I'm trying to run series_analysis_wrapper in met/10.0.0 on Orion with the modules listed on https://dtcenter.org/community-code/metplus/metplus-4-0-existing-builds
But it shows error message that it can't import the netCDF4 properly as below.
"08/26 19:10:03.582 metplus (command_builder.py:255) ERROR: (series_analysis_wrapper.py:74) There was a problem importing modules: No module named 'netCDF4'"
Do you have any idea about this?
Thank you.

**Comment by jprestop** [comment]
Hi @weiwilliam.

I see you are having trouble running the Series Analysis Wrapper in METplus on Orion.  The problem you are encountering is happening because the Python module "intelpython3/2020.2" on Orion does not have the netCDF4 module available for use:

```
Orion-dtn-1[2] jprestop$ module load contrib
Orion-dtn-1[3] jprestop$ module load intel/2020.2
Orion-dtn-1[4] jprestop$ module load intelpython3/2020.2
Orion-dtn-1[5] jprestop$ python
Python 3.7.7 (default, Jun 26 2020, 05:10:03) 
[GCC 7.3.0] :: Intel(R) Corporation on linux
Type "help", "copyright", "credits" or "license" for more information.
Intel(R) Distribution for Python is brought to you by Intel Corporation.
Please check out: https://software.intel.com/en-us/python-distribution
>>> import netCDF4
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'netCDF4'
```

If this is something you would like to use, you would need to contact the Orion helpdesk and ask them to install it for that module so that it is available for use.

**Reply by weiwilliam** [reply]
Thank you, @jprestop
Same as what I found, I'll ask them for help.

**Reply by weiwilliam** [reply]
In case other users on Orion encounter the same question, the solution provided by Orion helpdesk is here.
They suggest user to install NetCDF python module/API themselves by following commands.
module load intelpython3/2020.2
python3 -m pip install --user netcdf4

**Comment by jprestop** [comment, marked as answer]
Thanks you for letting us know the solution (below) from the Orion helpdesk!

> They suggest user to install NetCDF python module/API themselves by following commands.
> module load intelpython3/2020.2
> python3 -m pip install --user netcdf4
